src/Umbraco.Core/Configuration/ClientDependencyConfiguration.cs

```diff
@@ -49,5 +53,57 @@ internal bool IncreaseVersionNumber()
 
             return false;
         }
+
+        /// <summary>
+        /// Clears the temporary files stored for the ClientDependency folder
+        /// </summary>
+        /// <param name="currentHttpContext"></param>
+        internal bool ClearTempFiles(HttpContextBase currentHttpContext)
+        {
+            var cdfTempDirectories = new HashSet<string>();
+            foreach (BaseCompositeFileProcessingProvider provider in ClientDependencySettings.Instance
+                .CompositeFileProcessingProviderCollection)
+            {
+                var path = provider.CompositeFilePath.FullName;
+                cdfTempDirectories.Add(path);
+            }
+
+            try
+            {
+                var relativePath = currentHttpContext.Server.MapPath(XmlFileMapper.FileMapVirtualFolder);
+                if (relativePath != null)
+                {
+                    var directoryInfo = new DirectoryInfo(relativePath);
+                    var path = directoryInfo.FullName;
+                    cdfTempDirectories.Add(path);
+                }
+            }
+            catch (Exception ex)
+            {
+                //invalid path format or something... try/catch to be safe
+                LogHelper.Error<ClientDependencyConfiguration>("Could not get path from ClientDependency.config", ex);
+            }
+
+            var noErrorsDeleting = true;

```

Comment:
would be a bit less confusing to just call this `success` instead of a double negative